docs/fedora-releases-guide.md

```diff
@@ -138,6 +138,27 @@ configuration key.
 
 ![Dashboard view for pull_from_upstream](img/fedora-releases-guide/pull-from-upstream-dashboard.png)
 
+#### Trying it out
+If you are interested in this functionality and want to try it out, you can trigger the job from a pull request.
+
+If there is a new release pending for your package ([release-monitoring.org](https://release-monitoring.org/) bugzilla has been opened but no rebase done in dist-git yet), do the following:
+
```

Comment:
```suggestion
If there is a new release pending for your package (Bugzilla created by [release-monitoring.org](https://release-monitoring.org/) has been opened, but no rebase has been done in dist-git yet), do the following:

```

Reply:
I don't like the "conflict" between created and opened here, how about "bugzilla has been opened by release-monitoring.org"?

Reply:
I missed that, sure, that makes sense